pyproject.toml

```diff
@@ -74,9 +74,8 @@ responses = "^0.22.0"
 pytest-asyncio = "^0.20.3"
 
 [tool.poetry.group.lint.dependencies]
-flake8-docstrings = "^1.6.0"
 isort = "^5.10.1"
```

Comment:
If you want, you can also remove `isort` here, remove the `[tool.isort]` section below, then add:

```toml
[tool.ruff]
select = [
  "E",  # pycodestyle
  "F",  # pyflakes
  "I",  # isort
]
```